codalab/lib/bundle_cli.py

```diff
@@ -695,11 +734,9 @@ def do_upload_command(self, args):
             raise UsageError('Nothing to upload')
 
         # Pull out the upload bundle type from the arguments and validate it.
-        if args.bundle_type not in UPLOADED_TYPES:
-            raise UsageError('Invalid bundle type %s (options: [%s])' % (
-              args.bundle_type, '|'.join(sorted(UPLOADED_TYPES)),
-            ))
-        bundle_subclass = get_bundle_subclass(args.bundle_type)
+        # Note: only allow dataset bundles (eventually deprecate the program bundle and just have )
+        bundle_type = 'dataset'
```

Comment:
Eventually, but for now, everything in the system is a dataset or program
bundle_type, so let's leave it that way to not break anything.

On Tue, Nov 10, 2015 at 12:06 AM, Stephen Koo <notifications@github.com>
wrote:

> In codalab/lib/bundle_cli.py
> <https://github.com/codalab/codalab-cli/pull/213#discussion_r44378748>:
>
> > @@ -695,11 +734,9 @@ def do_upload_command(self, args):
> >              raise UsageError('Nothing to upload')
> >
> >          # Pull out the upload bundle type from the arguments and validate it.
> > -        if args.bundle_type not in UPLOADED_TYPES:
> > -            raise UsageError('Invalid bundle type %s (options: [%s])' % (
> > -              args.bundle_type, '|'.join(sorted(UPLOADED_TYPES)),
> > -            ))
> > -        bundle_subclass = get_bundle_subclass(args.bundle_type)
> > +        # Note: only allow dataset bundles (eventually deprecate the program bundle and just have )
> > +        bundle_type = 'dataset'
>
> Should we potentially just set this to 'uploaded' (and add the
> appropriate fields in UploadedBundle to make it a concrete type)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/213/files#r44378748>.
>